src/flask_track_usage/__init__.py

```diff
@@ -199,7 +199,10 @@ def after_request(self, response):
             data['ip_info'] = ip_info
 
         for storage in self._storages:
-            storage(data)
+            try:
+                storage(data)
+            except:
+                pass
```

Comment:
Question: Should we log at this point or is `pass` fine from a user perspective?

Reply:
Yeah, we could log them.  This may need rework.